### ISSUE TITLE
Editor does not properly load `PackedScene` in binary form when imported through an EditImportPlugin

### ISSUE THREAD

**Issue by Rubonnek** [godotengine/godot#64280]
### Godot version

3.x at 850c5ccccff0b14b3de8a287bbc803a9523780b6

### System information

Arch Linux

### Issue description

The Editor does not properly load `PackedScene` in binary form when imported through an EditImportPlugin.

Caused by #62408

### Steps to reproduce

1. Download and open MRP below.
2. Click "Open anyway".
3. You'll get the following messages in the output:
```
ERROR: Resource file not found: res://logo_map.tmx::1.
   at: _load (core/io/resource_loader.cpp:288)
WARNING: Couldn't load resource: res://logo_map.tmx::1
     at: parse_variant (core/io/resource_format_binary.cpp:298)
```
4. In the editor you are supposed to see the Godot logo within the TileMap but nothing shows up. When you run the scene, the logo shows up however.
----
As a side note, if you tweak the Tiled Map importer to save the `PackedScene` in text form by modifying `line 40` of `addons/vnen.tiled_importer/tiled_import_plugin.gd` to use `tscn` instead of `scn`, the error messages above will not appear after reimporting the Tiled Map and the Editor will properly show the Godot logo.

### Minimal reproduction project

[Binary_Scene_Bug.zip](https://github.com/godotengine/godot/files/9309939/Binary_Scene_Bug.zip)

**Comment by Rubonnek** [comment]
CC @Razoric480

**Comment by Razoric480** [comment]
Thanks, will take a look tonight.

**Comment by akien-mga** [comment]
Fixed by #64298.
